tests/core/v5_1/test_dispatcher_session_management.py

```diff
@@ -1,29 +1,60 @@
+import secrets
+
 from eth_enr import OldSequenceNumber
 import pytest
 import trio
 
+from ddht.base_message import AnyOutboundMessage
 from ddht.v5_1.constants import SESSION_IDLE_TIMEOUT
+from ddht.v5_1.messages import PingMessage
 
 
 @pytest.mark.trio
-async def test_dispatcher_intiator_and_recipient_detects_handshake_timeout(
-    driver, autojump_clock
+async def test_dispatcher_detects_handshake_timeout_as_initiator(
+    alice, bob, alice_client, bob_client, autojump_clock
 ):
-    await driver.initiator.send_ping()
+    await bob_client.manager.stop()
+    message = AnyOutboundMessage(
+        PingMessage(
+            secrets.token_bytes(4), bob_client.enr_manager.enr.sequence_number,
+        ),
+        bob.endpoint,
+        bob.node_id,
+    )
+    await alice_client.dispatcher._outbound_message_send_channel.send(message)
```

Comment:
I assume this line is to trigger creation of the new session.  If so, I think it would be better to directly call `alice_client.pool.initiate_session` (or `alice_client.pool.receive_session`).